.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "es/browser",
+  "globals": {
+    "React": true,
+    "ReactDOM": true
```

Comment:
Tatsächlich wundert es mich, dass das notwendig ist – bei uns ist es das nicht. Aber ich kann Dir gerade nicht erklären, warum …

Reply:
Hm komisch... liegt das wieder an meiner besondren VS-Code Config? 
Oder kannst du es - in diesem Repo - bei dir nachstellen? (Bin mir gerade unsicher, ob due mit "bei uns ist es das nicht" in einem anderen Repo wie Wolkenkit meinst oder tatsächlich hier)